lib/error_tracker/migration/sqlite/v05.ex

```diff
@@ -0,0 +1,17 @@
+defmodule ErrorTracker.Migration.SQLite.V05 do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  def up(_opts) do
+    alter table(:error_tracker_errors) do
+      add :muted, :boolean, default: false, null: false
+    end
+  end
+
+  def down(_opts) do
+    alter table(:error_tracker_errors) do
+      remove :muted
```

Comment:
This will not work, as SQLite does not support removing columns. We can leave it as it is  or emit an exception, as your wish 😅

Reply:
So it looks like this was not supported in old SQLite versions but it has been [supported](https://sqlite.org/lang_altertable.html#altertabdropcol) for a while.

I just tried in my local environment (added a V06 SQLite migration that did this) and can confirm that it works fine now.